grakn-core/src/main/java/ai/grakn/exception/GraqlQueryException.java

```diff
@@ -146,7 +150,7 @@ public static GraqlQueryException insertMetaType(Label label, SchemaConcept sche
 
     /**
      * Thrown when a concept is inserted with multiple properties when it can only have one.
-     *
+     * <p>
```

Comment:
Have a 🐐  to mark this incorrect use of a `<p>` tag. I.e. it has no closing tag.